src/errorReporting.js

```diff
@@ -126,7 +126,11 @@ function thenSendErrorReport({
           },
         }).then((response) => {
           if (response.ok) {
-            return response.json();
+            const textResponse = response.clone();
+            const jsonResponse = response.clone();
+            return jsonResponse.json()
+              .catch(() => textResponse.text())
+              .then((data) => data);
```

Comment:
any reason for the extra `.then`
```suggestion
              .catch(() => textResponse.text());
```

Reply:
also wondering if this has a purpose

Reply:
Resolved in c06ac75